app/helpers/jobs_helper.rb

```diff
@@ -0,0 +1,38 @@
+module JobsHelper
+
+  def delegate(params)
+    case
+    when params[:sort] == 'interest'
+      @jobs = Job.order(:level_of_interest)
+      render :interest_sorted
+    when params[:sort] == 'location'
+      @jobs = Job.order(:city)
```

Comment:
i would push this down further to the job model and use references to those methods